rusk-wallet/src/wallet.rs

```diff
@@ -385,6 +379,19 @@ impl<F: SecureWalletFile + Debug> Wallet<F> {
         derive_phoenix_vk(seed, index)
     }
 
+    /// get cache database path
+    pub(crate) fn db_path(&self) -> Result<PathBuf, Error> {
```

Comment:
Why call it `db_path` when all the variables and wrapper fn use `cache` instead of `db`?

Reply:
I think the db prefix is fine the rocksdb database we use is DBCommon

Reply:
but we in the wallet call the `db` cache, so in this context it's the `cache_path`

Reply:
renamed to `cache_path`